src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -2,9 +2,13 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.function.Predicate;
+
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
+import seedu.address.model.person.ItemContainsKeywordsPredicate;
```

Comment:
Nice to see that we check the item inside person as well as their name. Placement of this predicate also makes sense👍